### DISCUSSION TITLE
Location unknown on web-client

### DISCUSSION THREAD

**Discussion by redispade** [shlinkio/discussions#62] [Help wanted] [answered]
#### Shlink web client version

* Version: latest
* How do you use shlink-web-client: app.shlink.io|Docker image|self-hosted - Tried all of them

#### Summary

Hi. I am having issues with displaying the location of the visits. I am using shlink on docker, and the docker webclient(tried both on the same host and another ip). I tried also with app.shlink.io and the issue is the same I didn't notice when it happened, but I remember having the locations working somewhere in May.

I am able to see the locations of the visits when I use the commandline on the shlink host:
docker-compose exec shlink shlink visit:non-orphan reproduces a table that includes the locations.
I am just unable to see them on the web client. It displays unknown. Is there anything I can try? 
Edit: I already tried going back up to v3.5.1 with the client, but it's the same. Probably it isn't even client related?

**Comment by redispade** [comment]
This is actually a duplicate of https://github.com/shlinkio/shlink/discussions/1461

**Reply by acelaya** [reply]
As explained there, this is not the web clients doing. It looks to me like some kind of caching (reverse proxy or similar).

**Reply by redispade** [reply]
I have replied on the other discussion with more details and a solution. You may close this

**Comment by acelaya** [comment, marked as answer]
Solution here https://github.com/shlinkio/shlink/discussions/1461#discussioncomment-3063253
